source/javascripts/controllers/_MainController.js.erb

```diff
@@ -674,11 +691,12 @@ import datadogRumCustomTiming from "../utils/datadogCustomRumTiming.ts";
 				launchDarklyService.initialize().then(() => {
 					viewModel.guidedOnboardingEnabled = launchDarklyService.variation('enable-guided-onboarding');
 					if (viewModel.guidedOnboardingEnabled) {
-						const metadataReq = requestService.getCurrentUserMetadata('show_getting_started_guide');
+						const guideMetadataReq = requestService.getCurrentUserMetadata('show_getting_started_guide');
+						const lastWFEDateMetadataReq = requestService.getCurrentUserMetadata('last_workflow_edited_date');
 						const tourReq = requestService.getProductTourShown();
-
-						Promise.all([metadataReq, tourReq]).then(([metadataResp, tourResp]) => {
-							viewModel.guidedOnboardingEnabledForUser = metadataResp;
+						Promise.all([guideMetadataReq, lastWFEDateMetadataReq, tourReq]).then(([guideMetadataResp,lastWFEDateMetadataResp, tourResp]) => {
+							viewModel.guidedOnboardingEnabledForUser = guideMetadataResp;
+							viewModel.lastWorkflowEditedDate = lastWFEADateMetadataResp;
```

Comment:
`lastWFEADateMetadataResp` is not defined, I think you have a typo and it should be `lastWFEDateMetadataResp`

Reply:
This has been updated